src/derivatives/atlas.md

```diff
@@ -522,18 +502,18 @@ A guide for using macros can be found at
    "ps13-with-atlases-pipeline": {
       "tpl-PS13": {
          "pet": {
-            "tpl-PS13_atlas-Economo1916_desc-nopvc_dseg.nii.gz": "",
-            "tpl-PS13_atlas-Economo1916_desc-pvc_dseg.nii.gz": "",
-            "tpl-PS13_atlas-Economo1916_dseg.json": "",
-            "tpl-PS13_atlas-Economo1916_dseg.tsv": "",
-            "tpl-PS13_atlas-RamonCajal1908_desc-nopvc_dseg.nii.gz": "",
-            "tpl-PS13_atlas-RamonCajal1908_desc-pvc_dseg.nii.gz": "",
-            "tpl-PS13_atlas-RamonCajal1908_dseg.json": "",
-            "tpl-PS13_atlas-RamonCajal1908_dseg.tsv": "",
-            "tpl-PS13_desc-nopvc_dseg.nii.gz": "",
-            "tpl-PS13_desc-pvc_dseg.nii.gz": "",
-            "tpl-PS13_dseg.json": "",
-            "tpl-PS13_dseg.tsv": "",
+            "atlas-Economo1916_desc-nopvc_dseg.nii.gz": "",
```

Comment:
I would suggest to keep the tpl entity added to the filename, so the file can be shared as a standalone entity. Alternatively, make it clear that the various atlases now belong to the space PS13 (i.e. add space-PS13)

Reply:
I'm not sure I understand why it can't be shared as a standalone entity as is?  Regarding `space` entity.  I guess we should decide if `space` is REQUIRED when using atlas, or if it's only REQUIRED for disambuguaiton.